index.bs

```diff
@@ -1388,8 +1388,17 @@ When this method is invoked, the user agent MUST execute the following algorithm
     |lifetimeTimer| to this adjusted value. If the {{PublicKeyCredentialCreationOptions/timeout}} member of |options| is [=present|not
     present=], then set |lifetimeTimer| to a [=client=]-specific default.
 
-        Note: A suggested reasonable range for the {{PublicKeyCredentialCreationOptions/timeout}} member of |options| is
-        15 seconds to 120 seconds.
+      * Recommended ranges for the {{PublicKeyCredentialCreationOptions/timeout}} member of |options| are as follows.
```

Comment:
I don't think this needs a lone bullet point? Just a paragraph would seem ok.

(Here and below.)

Reply:
agree w/@agl

Reply:
Changed